src/SprykerEco/Shared/Heidelpay/Transfer/heidelpay.transfer.xml

```diff
@@ -58,7 +58,7 @@
     </transfer>
 
     <transfer name="HeidelpayCreditCardPaymentOptions">
-        <property name="optionsList" type="HeidelpayPaymentOption[]"/>
+        <property name="optionsList" type="HeidelpayPaymentOption[]" singular="optionsList"/>
```

Comment:
Singular is invalid as well as in transfers below.

Reply:
Fixed.